.env.example

```
@@ -8,3 +8,5 @@ VITE_OAUTH2_REDIRECT_URI=http://localhost:5173/login
 VITE_OAUTH2_SCOPE=openid
 VITE_OAUTH2_PROMPT=consent
 
+# Resource Center URL
+VITE_RESOURCE_CENTER_BASE=https://your-resource-center-url
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)